core/main/src/firebolt/handlers/parameters_rpc.rs

```diff
@@ -93,8 +94,19 @@ impl ParametersServer for ParametersImpl {
             .send_app_request(app_request);
         let resp = rpc_await_oneshot(app_resp_rx).await?;
         if let AppManagerResponse::LaunchRequest(launch_req) = resp? {
+            let country_code =
+                StorageManager::get_string(&self.platform_state, StorageProperty::CountryCode)
+                    .await
+                    .unwrap_or_default();
+
+            let us_privacy = if country_code == "US" {
+                privacy_data.get(privacy_rpc::US_PRIVACY_KEY).cloned()
+            } else {
+                None
+            };
+
             return Ok(AppInitParameters {
-                us_privacy: privacy_data.get(privacy_rpc::US_PRIVACY_KEY).cloned(),
+                us_privacy,
```

Comment:
What about `us_privacy` values in `badger_info` call and `AllowAppContentAdTargetingSettings` ?